openshift/kustomize/elastic/base/statefulset.yaml

```diff
@@ -121,7 +121,7 @@ spec:
         accessModes:
           - ReadWriteOnce
         storageClassName: netapp-block-standard
-        persistentVolumeReclaimPolicy: Delete
+        persistentVolumeReclaimPolicy: Retain
```

Comment:
I don't know if this is correctly, but it appears to be what we would want when the PVC gets deleted.